src/services/apis.rs

```diff
@@ -0,0 +1,52 @@
+use serde_json::Value;
+
+use crate::{
+    models::{ListKeysRequest, ListKeysResponse},
+    routes::{self, CompiledRoute},
+    services::HttpService,
+    types::Response,
+    unwind_response,
+};
+
+#[allow(unused_imports)]
+use crate::types::HttpError;
+
+#[derive(Debug, Clone)]
+pub struct ApiService;
+
+impl Default for ApiService {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl ApiService {
+    #[must_use]
```

Comment:
Documentation missing, but is very simple here - no example needed as the end user isn't necessarily supposed to be instantiating these service structs anyways.